paper/paper.md

```diff
@@ -48,67 +48,67 @@ bibliography: paper.bib
 To date, about 25% of anthropogenic carbon emissions have been taken up by the ocean [@Friedlingstein2022]. 
 This occurs through complex interactions between physics, chemistry, and biology, much of which is poorly understood. 
 Due to the vast size of the ocean and the sparsity of data; modelling and data assimilation play a vital role in quantifying the ocean carbon cycle. 
-Traditionally ocean biogeochemical (BGC) modelling involves large and inflexible code bases written in high-performance but low-level languages which require huge computational resources to execute. 
-Modern ocean dynamics models (e.g. Oceananigans.jl) ease the difficulty as they allow easy coupling with the physics components, but still require the user to implement all of the necessary BGC tools from the ground up.
-This causes a barrier to experimentation and innovation as users must develop expertise in both the science and complex code.
+Traditionally ocean biogeochemical (BGC) modelling involves large and inflexible code bases written in high-performance but low-level languages. 
+Using and modifying these models usually requires knowledge of the details of the numerical implementation (e.g. the grid layout and time-stepping schemes). 
```

Comment:
```suggestion
Both using and also modifying these models usually requires knowledge of the details of the numerical implementation (e.g. the grid layout and time-stepping schemes). 
```

reads better? not sure... feel free to ignore

Reply:
I kind of like the original here

Reply:
okie!